lib/actionkit_connector/client.rb

```diff
@@ -41,5 +41,21 @@ def basic_auth
     def base_uri
       URI.join(host, '/rest/v1/').to_s
     end
+
+    def pager_for(uri)
+      Enumerator.new do |y|
```

Comment:
`y` as a name is pretty standard, since the block to a new enumerator always receives a yielder object. I'll update anyway to make it clearer.

You can see it's usage here: https://github.com/SumOfUs/Champaign/blob/development/lib/tasks/action_kit.rake#L17-L19